README.md

```diff
@@ -367,7 +367,8 @@ values (EG: "default_server": True)
                                 "share_adjusters": [
                                     {
                                         "share_adjuster_class": "aurproxy.share.adjusters.RampingShareAdjuster",
```

Comment:
Related to fixing the RampingShareAdjuster documentation: I think that the share_adjuster_class needs to be "tellapart.aurproxy.share.adjusters.RampingShareAdjuster". I can make the change in a future pull request if you don't want to add it to this one.